roles/varnish/templates/etc/varnish/varnish.vcl

```diff
@@ -176,15 +176,17 @@ sub vcl_recv {
 
     # cnx rewrite archive  - specials served from nginx statically
     if (req.http.host ~ "^{{ arclishing_domain }}" || req.url ~ "^/sitemap.*.xml") {
-        if ( req.method == "POST" || req.method == "PUT" || req.method == "DELETE" || req.url ~ "^/(publications|callback|a|login|logout|moderations|feeds/moderations.rss|contents/.*/(licensors|roles|permissions))") {
+        
+        if (req.url  == "/robots.txt" || req.url ~ "^/specials") {
```

Comment:
No regex anchor needed for robots?

I seem to remember this block having moved before.

Reply:
I made it `==` since only the one URL is valid

Reply:
As to the move, this just seemed neater to me. Take care of the simple cases, leave the complex one to later